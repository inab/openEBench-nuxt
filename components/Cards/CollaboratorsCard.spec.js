describe('CollaboratorsCard', () => {
	test.todo('should be implemented');
});
