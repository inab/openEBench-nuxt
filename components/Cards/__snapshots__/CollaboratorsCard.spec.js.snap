// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollaboratorsCard should match snapshot 1`] = `
"<v-card-stub loaderheight=\\"4\\" href=\\"https://projectname.com\\" tag=\\"div\\" target=\\"_blank\\" maxheight=\\"350\\" maxwidth=\\"344\\" minheight=\\"250\\" hover=\\"true\\" class=\\"mx-auto mb-5\\">
  <div class=\\"image-container\\">
    <v-img-stub maxwidth=\\"80%\\" options=\\"[object Object]\\" position=\\"center center\\" src=\\"https://via.placeholder.com/150\\" transition=\\"fade-transition\\" class=\\"responsive-img\\"></v-img-stub>
  </div>
  <v-divider-stub></v-divider-stub>
  <h3 class=\\"text-center mt-3\\">Project Name</h3>
</v-card-stub>"
`;
